$MirOS: ports/net/curl/patches/patch-lib_ftp_c,v 1.1 2008/04/05 21:54:39 tg Exp $
--- lib/ftp.c.orig	Wed May  7 21:01:58 2008
+++ lib/ftp.c	Tue Aug 26 19:03:21 2008
@@ -2008,7 +2008,7 @@ static CURLcode ftp_state_mdtm_resp(stru
                  Curl_wkday[tm->tm_wday?tm->tm_wday-1:6],
                  tm->tm_mday,
                  Curl_month[tm->tm_mon],
-                 tm->tm_year + 1900,
+                 (int)(tm->tm_year + 1900),
                  tm->tm_hour,
                  tm->tm_min,
                  tm->tm_sec);
